### ISSUE TITLE
Tab focus

### ISSUE THREAD

**Issue by cbayerlein** [mastodon/mastodon#11319]
### Expected behaviour

In the Webapp, the tab focus should be visible - this is an accessibility issue and especially important for people who dependant on keyboard input.

**Comment by Kjwon15** [comment]
Yes, it is too barely visible currently.
